codice/ddf
# DDF-4460 Display name for metacard and query response transformers in UI

#### What does this PR do?
- Adds display names to export actions
- Sorts query response transformer list

#### Who is reviewing it? 

@Corey-Collins 
@leo-sakh 

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@djblue
@jrnorth 

#### How should this be tested?

- Verify that transformer display names are present in export actions drop down and result set action drop down. 
- Verify that export actions for query results function as expected. 
- Verify that exporting result sets from the table visual works as expected. 

#### Any background context you want to provide?
- Enhances user experience because human readable names are displayed instead of transformer IDs. 

#### What are the relevant tickets?
[DDF-4460](https://codice.atlassian.net/browse/DDF-4460)

#### Screenshots

Before
<img width="303" alt="screen shot 2019-01-24 at 2 43 57 pm" src="https://user-images.githubusercontent.com/7646577/51710477-8e309b00-1fe6-11e9-9e6d-2955e14517d3.png">
<img width="1217" alt="screen shot 2019-01-24 at 2 44 09 pm" src="https://user-images.githubusercontent.com/7646577/51710478-8e309b00-1fe6-11e9-9d56-06c3725cc46a.png">

After
<img width="297" alt="screen shot 2019-01-24 at 7 47 45 am" src="https://user-images.githubusercontent.com/7646577/51710322-00ed4680-1fe6-11e9-8204-4ce88721be1e.png">
<img width="1403" alt="screen shot 2019-01-24 at 8 12 25 am" src="https://user-images.githubusercontent.com/7646577/51710323-0185dd00-1fe6-11e9-87f7-bf3cd7e876c2.png">

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
